packages/react-core/src/components/Button/examples/Button.md

```diff
@@ -14,70 +14,103 @@ import ArrowRightIcon from '@patternfly/react-icons/dist/esm/icons/arrow-right-i
+| Link | Links are labeled, but have no background or border. Use for actions that require less emphasis, actions that navigate users to another page within the same window, and/or actions that navigate to external pages in a new window. Links may be placed inline with text using the `isInline` prop.|
+| Plain | Plain buttons have no styling and are intended to be labeled with icons. |
+| Control | Control buttons can be labeled with text or icons. Primarily intended to be paired with other controls in an [input group](https://www.patternfly.org/v4/components/input-group). | 
+
+### Disabled buttons
+
+To indicate that an action is currently unavailable, all button variations can be disabled using the `isDisabled` prop.
 
 ```ts file="./ButtonDisabled.tsx"
 ```
 
-### Aria disabled
+### Progress indicators
 
-```ts file="./ButtonAriaDisabled.tsx"
+Progress indicators can be added to buttons to identify that an action is in progress after a click. 
+
+```ts file="./ButtonProgress.tsx"
 ```
 
-### Aria disabled button with tooltip
+### Small buttons
 
-```ts file="./ButtonAriaDisabledTooltip.tsx"
+To fit into tight spaces, primary, secondary. tertiary, danger, and warning button variations can be made smaller using the `isSmall` prop.
+
+```ts file="./ButtonSmall.tsx"
 ```
 
-### Aria disabled link as button with tooltip
+### Call to action (CTA) buttons
 
-```ts file="./ButtonAriaDisabledLinkTooltip.tsx"
+CTA buttons and links direct users to complete an action. Primary, secondary, tertiary, and link button variants can be styled as CTAs using the `isLarge` prop. 
+
+```ts file="./ButtonCallToAction.tsx"
+```
+
+### Block level buttons
+
+Block level buttons span the full width of the parent element and can be enabled using the `isBlock` prop.
+
+```ts file="./ButtonBlock.tsx"
 ```
 
+## Links 
+
 ### Links as buttons
 
+Buttons that link to another resource may take the form of primary, secondary, tertiary, or link variants. Use `component="a"` and an `href` prop to designate the button's target link.
+
 ```ts file="./ButtonLinks.tsx"
 ```
 
 ### Inline link as span
 
+Inline links should use `component="span"` and the `isInline` prop to wrap inline with surrounding text.
+
 ```ts file="./ButtonInlineSpanLink.tsx"
 ```
 
-### Block level
-
-```ts file="./ButtonBlock.tsx"
-```
+### Router link
 
-### Types
+Router links can be used for in-app linking in React environments to prevent page reloading.
 
-```ts file="./ButtonTypes.tsx"
+```ts file="./ButtonRouterLink.tsx"
 ```
 
-### Small
+## Aria buttons
 
-```ts file="./ButtonSmall.tsx"
-```
+Accessible Rich Internet Applications (ARIA) is a set of roles and attributes specified by the [World Wide Web Consortium](https://www.w3.org/WAI/standards-guidelines/aria/). ARIA defines ways to make web content and web applications more accessible to people with disabilities.
 
-### Call to action
+### Aria disabled buttons
```

Comment:
It feels redundant to have all three examples state `Aria disabled button...`, but I'm not sure if titling the subsection `Aria disabled button` would make it clear enough to adjust the repeated example title text. Maybe `Using aria disabled` for the section title? Anyone have thoughts about these names?

Reply:
That's a fair point I think, and is worth aligning on now since there are other components that have/may have examples laid out like this. I feel like if the section was renamed to "Aria-disabled button", the examples could be renamed to something along the lines of "Basic aria-disabled", "With tooltip", and if we still wanted the example, "Link as button with tooltip". The heading levels plus the way the table of contents is laid out should provide the context that these three examples are for "aria disabled buttons".

The only issue with that might be if there were a "basic with tooltip" and "aria-disabled with tooltip" example on a component since we couldn't have two named "with tooltip", but then I think just naming them both as "[type] with tooltip" would work